libraries/wallet/wallet.cpp

```diff
@@ -2258,6 +2258,8 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
             SET_PROP(p, min_delegation);
             op.props = p;
             auto hf = my->_remote_database_api->get_hardfork_version();
+            chain_properties_19 p19;
+            chain_properties_21 p21;
             if (hf >= hardfork_version(0, STEEMIT_HARDFORK_0_19) || !!props.max_referral_interest_rate
```

Comment:
You can simplify this part of code - there are no more nodes with HF-18:
`chain_properties_18 p;` - > `chain_properties_19 p;`